.idea/misc.xml

```diff
@@ -49,7 +56,7 @@
   <component name="ProjectResources">
     <default-html-doctype>http://www.w3.org/1999/xhtml</default-html-doctype>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" assert-keyword="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
@motlin ouch, sorry, in fact I didn't meant to commit any IDE files and was quite annoyed that they were showing up as modified (maybe relates to #199), so just overlooked this change - let me revert IDE files back